README.md

```diff
@@ -32,17 +32,17 @@ Please read, understand and adhere to our [code of conduct](CODE_OF_CONDUCT.md).
 
 ## Install
 
-To download pre-built binaries, see our [downloads page](https://tendermint.com/intro/getting-started/download).
+To download pre-built binaries, see our [downloads page](https://tendermint.com/download).
```

Comment:
that redirects to /downloads so perhaps, let's replace
https://tendermint.com/downloads with https://tendermint.com/downloads?